tests/util/browserOperations.js

```diff
@@ -13,7 +13,7 @@ const taikoHelper = require("../util/taikoHelper")
 const headless = process.env.headless_chrome.toLowerCase() === 'true';
 
 beforeSuite(async () => {
-    await openBrowser({headless:headless, args:["--no-sandbox","--start-maximized","--disable-dev-shm-usage","--start-fullscreen",'--use-fake-ui-for-media-stream']})
+    await openBrowser({headless:headless, args:["--no-sandbox","--disable-dev-shm-usage",'--use-fake-ui-for-media-stream',"--window-size=1440,900"]})
```

Comment:
Can we run tests for tablet resolutions like this too? If so, we could have different modes to run all E2E tests in Browser mode and then in tablet mode?

Reply:
yes, we can run it with small resolution but our e2e scripts should be updated with updated selectors in most of the places, Like if we have selectors like toLeftOf or toRightOf then in lower resolution we may need to use below or above selectors.

Reply:
Can we do this for Lab-Lite, and see how much effort it is? You can plan this post MVP also. Then we can see if volunteer QAs can do this for other scenarios.

Reply:
Sure, Will Create a card in Lab Lite.

Reply:
https://bahmni.atlassian.net/browse/BAH-2058